pymeasure/instruments/nf/li5660.py

```diff
@@ -0,0 +1,1665 @@
+        """ A property that allow user to set the calculation method for measurement value to be displayed and output. """
+        log.info("The calculation method for measurement value is set to {}".format(self.name))
+
+        self.calculation_method = method
+
+    def buffers(self, buffer):
+        buffer_list = {"Buffer1": "BUF1", "Buffer2": "BUF2", "Buffer3": "BUF3"}
+        return buffer_list[buffer]
+
+    def get_buffer(self, buffer="Buffer1", lenght=1, start=0):
+        """ A aquire the floating point data through binary transfer. """
+        buffers = {"Buffer1": "BUF1",
+                  "Buffer2": "BUF2",
+                  "Buffer3": "BUF3"}
+        return self.values(":DATA:DATA? {}, {}, {}".format(buffers[buffer], lenght, start))
+
+    def clere_buffer(self, buffer="Buffer1"):
+        """ Clears the specified measurement data buffer. """
+        buffers = {"Buffer1": "BUF1",
+                  "Buffer2": "BUF2",
+                  "Buffer3": "BUF3"}
+        self.write(":DATA:DEL {}".format(buffers[buffer]))
+
+    def clear_all_buffer(self):
+        """ Clear all measurement data buffers. """
+        self.write(":DATA:DEL:ALL")
+
+    def data_feed(self, buffer="Buffer1", data=30):
+        """ Set measurement data sets recorded in measurement data buffer 1, 2 or 3. """
+        buffers = {"Buffer1": "BUF1",
+                  "Buffer2": "BUF2",
+                  "Buffer3": "BUF3"}
+        self.write(":DATA:FEED {}, {}".format(buffers[buffer], data))
+
+    def data_feed_control(self, buffer="Buffer1", state="Always"):
+        """ Set whether measurement data is to be recorded in a measurement data buffer. """
+        buffers = {"Buffer1": "BUF1",
+                  "Buffer2": "BUF2",
+                  "Buffer3": "BUF3"}
+        status = {"Always": "ALW",
+                  "Never": "NEV"}
+        self.write(":DATA:FEED:CONT {}, {}".format(buffers[buffer], status[state]))
+
+    def data_points(self, buffer="Buffer1", size=16):
+        """ Set measurement data buffer size. """
+        buffers = {"Buffer1": "BUF1",
+                  "Buffer2": "BUF2",
+                  "Buffer3": "BUF3"}
+        self.write(":DATA:POIN {}, {}".format(buffers[buffer], size))
+
+    def interval(self, time=10E-3):
+        """ A property that allow user to set the internal timer time interval. """
+        self.data_timer = time
+
+    def timer_state(self, state="Off"):
+        """ A property that allow user to enable or disable the internal timer. """
+        self.data_timer_state = state  
+
+    def default_display(self):
+        """ Configures the measurement screen. """
+        self.screen = "Fine"
+
+    def display(self, display=None):
+        """ A property that allows user to switch between the 'Standard', 'Expanded', 'Detail' screens. """
+        log.info("Screen is display in {}".format(self.name))
+        
+        if display is None:
+            self.default_display()
```

Comment:
Seems like a bit of a convoluted way to implement a default; what is the advantage over setting `'Fine'` as the default value for the `display` argument (in stead of calling a `default_display` method)?